snippets/csharp/concepts/linq/how-to-order-the-results-of-a-join-clause_1.cs

```diff
@@ -66,7 +66,7 @@ select prod2
                         Console.WriteLine(productGroup.Category);
                         foreach (var prodItem in productGroup.Products)
                         {
-                            Console.WriteLine("  {prodItem.Name:-10} {prodItem.CategoryID}");
+                            Console.WriteLine($"  {prodItem.Name:-10} {prodItem.CategoryID}");
```

Comment:
this should be `{prodItem.Name,-10}`.

Reply:
@rpetrusha that was a catch of the week! Thank you, fixed.